discuss.php

```diff
@@ -377,6 +377,7 @@
 </script>";
 
 $PAGE->requires->js_call_amd('mod_hsuforum/mod_hsuforum_focus', 'init');
-$PAGE->requires->js_call_amd('mod_hsuforum/mod_hsuforum_loader', 'init');
+$config = get_config('hsuforum');
+$PAGE->requires->js_call_amd('mod_hsuforum/mod_hsuforum_loader', 'init', ['enablePostSpinner' => $config->enablepostspinner]);
```

Comment:
please set a variable here. Set it to false and then do a !empty check on $config->enablepostspinner and change that variable accordingly and then pass that variable into you init.

Reply:
Happy with change